### PR TITLE
Add Jest coverage GitHub action to show Coverage Report in Pull Requests

### PR DESCRIPTION
**Pull request by thebearingedge**
This PR partially addresses #175. I'm not sure how you'd like to make the badge, but this will show coverage for new PRs.

PS: Your project is awesome! 🤓

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,4 +30,11 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: npm run lint
       - run: npm run build
-      - run: npm test
+      - name: Run Tests on Node.js ${{ matrix.node-version }}
+        if: ${{ matrix.env.COVERAGE != '1' }}
+        run: npm test
+      - name: Run Tests w/Coverage on Node.js ${{ matrix.node-version }}
+        if: ${{ matrix.env.COVERAGE == '1' }}
```
Comment (metrue) [inline .github/workflows/ci.yml:37]: I think coverage can be enabled by default, any concerns?

Comment (thebearingedge) [reply .github/workflows/ci.yml:37]: I guess I just didn't expect the outcome of running coverage for both versions of Node.js (redundant info).

Comment (metrue) [reply .github/workflows/ci.yml:37]: I see, coverage on one version should be enough.

**Comment by yusukebe** [comment]
Hi @thebearingedge !

Thank you! That's what I want. But, this GitHub Actions does not work well.
Can you fix it?

PS.
To make the badge on README, the command seems good:
https://github.com/jaywcjlove/coverage-badges-cli

**Comment by thebearingedge** [comment]
@yusukebe, hello! I should have known, but it looks like we will always have the same problem with Pull Requests from forks. 

I should have realized it, but write access for fork Pull Request actions is dangerous, so maximum token permissions will be **read**.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

I will see if there is another way some time tomorrow, but I think an external service or proxy app with be required.

😕 sorry
Screen Shot 2022-07-15 at 8 26 36 AM

**Comment by yusukebe** [comment]
@thebearingedge 

I see, but don't be sorry!
This is a complex problem. I think maybe we don't have to force it.

**Comment by thebearingedge** [comment]
closing for now

**Pull request closed without merging**